examples/Demo/Shared/Pages/Card/Examples/CardMinimalStyleAreaRestricted.razor

```diff
@@ -0,0 +1,42 @@
+﻿@inject DataSource Data
```

Comment:
I found it preferable to keep the `CardMinimalStyle.razor` example, which features the `MinimalStyle` attribute.
Correcting this problem should not result in a new example of use.
I find the new example too complex to explain functionality: it demonstrates functions that have nothing to do with each other.

Reply:
Yes hard to demonstrate `MinimalStyle`  
I used the new one test to make sure I didn't break it.   Reverted.